app/models/manageiq/providers/ibm_terraform/inventory/persister/configuration_manager.rb

```diff
@@ -9,7 +9,7 @@ def initialize_inventory_collections
   end
 
   def add_cross_provider_vms
-    add_collection(configuration, :vms) do |builder|
+    add_collection(configuration, :vms, {}, {:without_sti => true}) do |builder|
```

Comment:
NOTE this is re-defined by a few providers, want to add a new collection to core for cross-referenced VMs in a follow-up so providers don't have to define it from scratch